src/code/bank0.asm

```diff
@@ -5901,7 +5901,7 @@ LoadRoom::
 .loadBankForOverworldRooms
     ; … and the overworld room index is >= $80…
     ldh  a, [hMapRoom]                            ; $322F: $F0 $F6
-    cp   $80                                      ; $3231: $FE $80
+    cp   IS_INDOOR                                ; $3231: $FE $80
```

Comment:
This checks for lower upper half of the overworld, as data for those is split 50/50 in 2 banks.